cfpb/sbl-frontend
# fix(upload): set defaults, multiply polling env by 1k, match designs

Some cleanup and a fix for the env vars `SBL_LONGPOLLING_DELAY_SECONDS` and `SBL_UPLOAD_FILE_SIZE_LIMIT_BYTES`.

## Changes

- after the load test Friday afternoon, set `SBL_UPLOAD_FILE_SIZE_LIMIT_BYTES` to 50 MB by default
- set `SBL_LONGPOLLING_DELAY_SECONDS` to `backoff` default
- multiply `SBL_LONGPOLLING_DELAY_SECONDS` by 1000 before it's used to convert seconds to milliseconds
- remove trailing `.0`, adjust precision between mb and gbs, and add a space between the size and label to match the designs
- add the new vars to env-guide and test.yaml
## How to test this PR

1. Try to upload using the defaults, note that the page says 50mb and uses exponential backoff to call `/latest`
2. Try setting `SBL_LONGPOLLING_DELAY_SECONDS` to 2 seconds and `SBL_UPLOAD_FILE_SIZE_LIMIT_BYTES` to `200000000` to see 2 GB and calls to `/latest` every 2 seconds

## Screenshots

![Screenshot 2024-06-29 at 9 39 56 PM](https://github.com/cfpb/sbl-frontend/assets/19983248/610c0291-c2dd-4615-bf59-4b3c2cf6e1ad)

![Screenshot 2024-06-29 at 9 39 44 PM](https://github.com/cfpb/sbl-frontend/assets/19983248/75d7a423-ad01-4591-abeb-8ce3e47dc2ae)
![Screenshot 2024-06-29 at 9 40 09 PM](https://github.com/cfpb/sbl-frontend/assets/19983248/12c70de6-be93-43fb-8f07-c34dea4dca8b)
